e2e/native-async-mock/yarn.lock

```diff
@@ -0,0 +1,11 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
+  version: 4
+
+"root-workspace-0b6124@workspace:.":
+  version: 0.0.0-use.local
+  resolution: "root-workspace-0b6124@workspace:."
+  languageName: unknown
+  linkType: soft
```

Comment:
Why did this get checked in? I believe the test suite generated it. I believe it should be .gitignored